### DISCUSSION TITLE
How to add scripts to gameobjects?

### DISCUSSION THREAD

**Discussion by ianfrancong** [piotrulos/MSCModLoader#196] [Help me] [answered]
I have multiple scripts in a project but I'm struggling to get them added as components to the gameobjects I'm adding in. I've tried adding the scripts within the unity project so I could add them as components to the unity3d bundle but I keep getting bombarded with errors. Can someone please help me with this?

Edit: I've sort of figured it out but I can't get the scripts to actually load into the game. Everytime I build the asset bundle and place it into the project and build the scripts just aren't there ingame.

**Comment by piotrulos** [comment]
Scripts aren't added to assetbundle, because assetbundle doesn't store code or assemblies

**Reply by ianfrancong** [reply]
Okay, if I can't add them to the prefab then where am I supposed to insert them? I can't figure out how to add the scripts to the gameobjects as components

**Reply by piotrulos** [reply, marked as answer]
Just use AddComponent in your script.
